### DISCUSSION TITLE
How to stop highlighting after accepted a suggestion

### DISCUSSION THREAD

**Discussion by jasmine-nguyen** [saghen/blink.cmp#290] [Q&A]
Hi, just wondering if this is how the plugin should behave before I raise a bug. Basically when I accept a suggestion, the accepted suggestion is highlighted and and highlighting stays on so it won't detect that I already move on to the next word, hence the next word's auto complete does not get triggered. 

For example in [this clip](https://imgur.com/a/ozelCvo), when I accepted suggestion "On", the cursor highlighted as soon as I type "Make", after "Make" I had to do a tab to get outside of the bracket, then move the cursor back inside to continue typing without the highlighting and the next word's autocomplete gets triggered.

Is there anyway I can disable this or that's how it works? Thank you.

I'm on Neovim  v0.10.2. Blink v0.5.1. Below is my config.

```
return {
	"saghen/blink.cmp",
	event = { "LspAttach", "InsertCharPre" },
	version = "v0.*",
	opts = {
		accept = { auto_brackets = { enabled = true } },
		highlight = {
			use_nvim_cmp_as_default = true,
		},
		keymap = { preset = "super-tab" },
		nerd_font_variant = "mono",
		windows = {
			autocomplete = {
				border = "single",
			},
			documentation = {
				auto_show = true,
				border = "single",
			},
		},
		trigger = {
			completion = {
				show_in_snippet = false,
			},
			signature_help = { enabled = true },
		},
	},
}
```

**Comment by saghen** [comment]
You're seeing the behavior of the built-in snippet expansion. Unfortunately, nothing we can do on the blink side
